digits/inference/job.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from . import tasks
+import digits.frameworks
+from digits.job import Job
+from digits.utils import override
+
+class InferenceJob(Job):
+    """
+    A Job that exercises the forward pass of a neural network
+    """
+
+    @override
```

Comment:
No problem. In general I wouldn't use `@override` for any "dunder" functions (i.e. `__init__`, `__str__`, `__getstate__`), but I wouldn't worry about changing it in this PR - it doesn't hurt anything.

There's really not much to these functions at all:
https://github.com/NVIDIA/DIGITS/blob/v3.1.0/digits/utils/__init__.py#L75-L99